registry.json

```diff
@@ -1,5 +1,20 @@
 [
 
+   {
+       "ticker": "APE",                                                       
+       "poolId": "3c947f2150c633fcd157b9b0fc42ca45d28bba6d06ca4edda09de568",  
+       "social": {              
+         "website":"https://adaape.com", 
+         "twitter": "TheADAApe",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "The_ADA_Ape_Troop",                                             
```

Comment:
I cant see this user on telegram... Please can you make sure this is the handle of an individual and not a group or channel?